src/app/static/js/editor/codemirror.js

```diff
@@ -9531,10 +9531,10 @@ TextareaInput.prototype.needsContentAttribute = false
 
 function fromTextArea(textarea, options) {
   options = options ? copyObj(options) : {}
-  options.value = textarea.value
-  if (!options.tabindex && textarea.tabIndex)
+  options.value = textarea ? textarea.value : ""
+  if (!options.tabindex && textarea && textarea.tabIndex)
```

Comment:
why do we need this change?

Reply:
@rtgdk I did this to clear up console warning messages displayed when the textarea is null.